src/Libraries/RevitNodesUI/RevitNodesUI.csproj

```diff
@@ -36,7 +36,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="DynamoUtilities">
-      <HintPath>$(DYNAMOAPI)\DynamoUtilities.dll</HintPath>
+      <HintPath>..\..\..\..\..\..\..\..\Program Files\Dynamo\Dynamo Core\1.2\DynamoUtilities.dll</HintPath>
```

Comment:
So what happened here is that when I opened the solution i was missing bunch of references to Dynamo dlls. When I re-pointed them to the proper location that of course updated all of the paths in the csproj. Any idea how to make my solution recognize the $(DYNAMOAPI) variable and fix the missing references without manually re-setting them?